code/frontend/MakeItSo/MakeItSo/MakeItSo.docc/Tutorials/Building a simple todo list UI.tutorial

```diff
@@ -0,0 +1,473 @@
+        
+        > Note: Keep in mind that you own the preview, so you can adjust it to your needs to make it easier to see the view you're working on in the context it will later be used in.
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-05-contentview-present-addreminder-04.swift) {
+          @Image(source: building-todo-ui-reminder-05-contentview-present-addreminder-04.png, alt: "")
+        }
+      }
+      
+      @Step {
+        Use the `sheet` view modifier to present the `AddReminderView` dialog. Bind the `isPresented` parameter to the `isAddReminderPresented` property you defined in an earlier step.
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-05-contentview-present-addreminder-05.swift) {
+          @Image(source: building-todo-ui-reminder-05-contentview-present-addreminder-04.png, alt: "")
+        }
+      }
+      
+      @Step {
+        Inside the `completion` closure of the `AddReminderView` view, take the `remninder` parameter and append the new reminder to the list of reminders.
+        
+        @Code(name: "ContentView.swift", file: building-todo-ui-reminder-05-contentview-present-addreminder-06.swift) {
+          @Image(source: building-todo-ui-reminder-05-contentview-present-addreminder-04.png, alt: "")
+        }
+      }
+      
+      @Step {
+        Use the live preview and try out this new functionality by tapping the _New Reminder_ button to add a new reminder to the list of reminders.
+        
+        @Video(source: building-todo-ui-reminder-05-contentview-present-addreminder-05.mp4)
+      }
+    }
+  }
+  
+  @Section(title: "Brush up the UI") {
+    @ContentAndMedia {
+      To finish this first chapter, refine the UI:
+      
+      * Make sure the `title` input field is active 
+      * Make sure the _Add_ button is only active if the reminder input field is not empty
+      * Add a _Cancel_ button
+      * Add a title to the dialog
+      
+      @Image(source: placeholder-image.png, alt: "")
+    }
+    
+    @Steps {
+      You might have noticed that the input field was not active when the dialog appears. This is inconvenient, especially on larger phones, when the input field might be hard to reach. In the next few steps, you will use SwiftUI's `@FocusState` property wrapper to place the cursor in the first input field when the dialog appears.
+      @Step {
+        Go back to the `AddReminderView.swift` file, and define an enum `FocusableField` whith a single case `title`.  
+        
+        @Code(name: "AddReminderView.swift", file: building-todo-ui-reminder-06-addreminder-brushup-01.swift, previousFile: building-todo-ui-reminder-04-addreminderview-10.swift)
+      }
+      
+      @Step {
+        Add a new private property `focusedField` as an optional `FocusedField` and mark it using the `@FocusState` property wrapper.  
```

Comment:
Maybe link to the `FocusState` documentation so SwiftUI–unfamiliar readers (like myself) recognize that this is an Apple API.
```suggestion
        Add a new private property `focusedField` as an optional `FocusedField` and mark it using the [`@FocusState`](https://developer.apple.com/documentation/swiftui/focusstate) property wrapper.  
```
For all I know though, this might be a really common property wrapper like `@State` so most readers would know without having to look at the linked documentation.

Side note, for the sake of completeness, it'd be neat to write a script that scans the tutorial for Apple API names, and then hyperlinks them to the docs for improved discoverability when reading.

Reply:
This could be a cool contribution to DocC if it doesn't already do that.